lms/djangoapps/commerce/urls.py

```diff
@@ -0,0 +1,12 @@
+"""
+Defines the URL routes for this app.
+"""
+
+from django.conf.urls import patterns, url
+
+from .views import PurchaseView
+
+urlpatterns = patterns(
+    '',
+    url(r'^purchase/$', PurchaseView.as_view(), name="purchase"),
```

Comment:
Considering what this endpoint is currently designed to do and the purpose we may want it to serve in the near future (i.e., ordering paid seats), I think it would be a good idea to align the name of this endpoint with the name of the endpoint it corresponds to on the ecommerce service. What do you think of changing the endpoint's name to `/orders/` and dropping references to "purchase"? This will give us consistent terminology to work with across the LMS and the ecommerce service, and will allow us to use this endpoint when attempting to GET information about orders.

Is APPEND_SLASH enabled on the LMS? If it is, we might get requests to `/orders/` without POST data. The endpoint as you've designed it will handle that case gracefully, but it's just something to consider.